composer/datasets/ade20k.py

```diff
@@ -345,3 +347,126 @@ def initialize_object(self, batch_size, dataloader_hparams) -> DataSpec:
                                                                         collate_fn=collate_fn,
                                                                         drop_last=self.drop_last),
                         device_transforms=device_transform_fn)
+
+
+@dataclass
+class ADE20kWebDatasetHparams(WebDatasetHparams, SyntheticHparamsMixin):
```

Comment:
I dont think these need to support the `SyntheticHparamsMixin`. We already have a normal dataset with synthetic support correct?

Reply:
Done (78bae50bbf903ed01d858cc4562073963476798d)